### PR TITLE
Allow location header name to be lowercase

### PR DESCRIPTION
**Pull request by wmouwen**
Web servers and clients tend to cast header names to uppercase or lowercase. Guzzle handles these differences by [casting the names to lowercase](https://github.com/guzzle/psr7/blob/7858757f390bbe4b3d81762a97d6e6e786bb70ad/src/MessageTrait.php#L53-L64) when you try to access them. However, it will only do so when you get a single header, not when you request the entire collection.

This PR uses the `getHeader` function instead of `getHeaders`. As a result, it doesn't matter whether the server returns the header as `Location` or `location`.

### CODE REVIEW

**Comment by KerZoli** [comment]
Hello! Any update regarding if this will be accepted? I can confirm the issue also. Thanks.

**Review by SamirBoulil — APPROVED** [review]

File: src/Api/ProductMediaFileApi.php
Comment (mmetayer) [inline src/Api/ProductMediaFileApi.php:135]: we could also use the `hasHeader()` method

**Review by mmetayer — APPROVED** [review]

File: src/Api/ProductMediaFileApi.php
Comment (wmouwen) [reply src/Api/ProductMediaFileApi.php]: I've updated the code to use the `hasHeader()` helper. Thanks!

**Comment by wmouwen** [comment]
Please note that I cannot merge this pull request as I lack write access to the repository.

**Comment by KerZoli** [comment]
Is there any chance to merge the pull request? Thanks.

**Comment by mmetayer** [comment]
@KerZoli  apologies, this PR kinda slipped off my mind :confused: 
@wmouwen I fixed some failing tests, the PR is ready, but I need you to sign our [Contributor Licence Agreement](https://www.akeneo.com/contributor-license-agreement/) before merging. Can you ping me when it's OK?

**Pull request closed without merging**
